zulipterminal/ui_tools/views.py

```diff
@@ -751,6 +751,24 @@ def keypress(self, size: urwid_Size, key: str) -> Optional[str]:
         return super().keypress(size, key)
 
 
+class InfoView(urwid.ListBox):
+    def keypress(self, size: urwid_Size, key: str) -> str:
+        if is_command_key('GO_BACK', key) \
+                or is_command_key(self.command, key):
+            self.controller.exit_popup()
```

Comment:
`self.controller` and `self.command` are just assumed to be present here; I think it would be clearer if we just had an initializer which specified the parameters it needed and were used at each call point.